src/components/CalendarContainer.ts

```diff
@@ -212,4 +230,26 @@ export default class CalendarContainer extends Component<CalendarContainerProps,
             });
         }
     }
+
+    public static validateProps(props: CalendarContainerProps): string {
+        let errorMessage = "";
+        if (props.onClickEvent === "callMicroflow" && !props.eventMicroflow) {
+            errorMessage = `On click event is set to 'Call a microflow' but no microflow is selected`;
+        } else if (props.onClickSlotEvent === "callMicroflow" && !props.slotMicroflow) {
+            errorMessage = `On click slot is set to 'Call a microflow' but no microflow is selected`;
```

Comment:
Use template strings when you have placeholders within `${ .. }`

Reply:
Dome